src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -21,8 +22,8 @@
     /**
      * Returns an add command string for adding the {@code person}.
```

Comment:
```suggestion
     * Returns an add command string for adding the {@code participant}.
```

Reply:
but this one is for Person though hahahaha